File: app/src/main/java/com/pluu/webtoon/model/RToon.java
package com.pluu.webtoon.model;

import io.realm.RealmObject;

/**
 * 웹툰 Model
 * Created by PLUUSYSTEM-NEW on 2015-11-11.
 */
public class RToon extends RealmObject {
	private String service;
	private String toonId;

	public RToon() { }

	public void setService(String service) {
		this.service = service;
	}

	public void setToonId(String toonId) {
		this.toonId = toonId;
	}

	public String getService() {
		return service;
	}

	public String getToonId() {
		return toonId;
	}

}

File: app/src/main/java/com/pluu/webtoon/model/RToon.kt
package com.pluu.webtoon.model

import io.realm.RealmObject

/**
 * 웹툰 Model
 * Created by pluu on 2017-05-02.
 */
open class RToon : RealmObject() {
    open var service: String? = null
    open var toonId: String? = null
}
